bestbook/templates/base.html

```diff
@@ -38,37 +57,31 @@ <h1 class="main-logo">
-            or <a href="/login">login</a> to
-            {% endif %}
+          <div class="navigation block-rec">
+            <a class="filter {{'selected' if request.path in ['/browse', '/'] else ''}}" href="/browse">Find</a>,
+
             <!-- <a class="filter {{'selected' if request.path in ['/ask', '/'] else ''}}" href="/ask">Request</a>, or -->
             <a class="filter {{'selected' if request.path in ['/ask'] else ''}}" href="https://forms.gle/FQGbrTNY3kYfPoAA6" target="_blank">Request</a>, or
-            <a class="filter {{'selected' if request.path == '/submit' else ''}}" href="/submit">Make</a> Recommendations.
-            <br>
-            {% if session.get('username') and is_admin(session.get('username')) %}
-              <a class="filter {{'selected' if request.path in ['/admin'] else ''}}" href="/admin">Debug</a>
-              <a class="filter {{'selected' if request.path in ['/admin/approve/recommendations'] else ''}}" href="/admin/approve/recommendations">Approve Recommendations</a>
-              <a class="filter {{'selected' if request.path in ['/admin/approve/requests'] else ''}}" href="/admin/approve/requests">Approve Requests</a>
-            {% endif %}
-          </span>
-          <span style="display: inline; float: right;">
-            {% if session.get('username') %}
-              Welcome {{session.get('username')}}! (<a href="/logout">logout</a>)
-            {% endif %}
-          </span>
+            <a class="filter {{'selected' if request.path == '/submit' else ''}}" href="/submit">Recommend</a> the Best Book on any <a href="/topics">Topic</a>
+          </div>
+
           {% include template %}
         </div>
 
         <footer style="text-align: center;">
-          <p><a href="https://thebestbookon.com">TheBestBookOn.com</a> &copy; 2020 is a non-profit <a href="https://github.com/Open-Book-Genome-Project/TheBestBookOn.com">open-source</a> experiment maintained by <a href="https://mek.fyi">mek</a> @ <a href="https://bookgenomeproject.org">Open Book Genome Project</a>.</p>
+          <p><a href="https://thebestbookon.com">TheBestBookOn.com</a> &copy; 2020 is a non-profit <a href="https://github.com/Open-Book-Genome-Project/TheBestBookOn.com">open-source</a> experiment maintained by the <a href="https://bookgenomeproject.org">Open Book Genome Project</a>.</p>
           <p>We gratefully borrow book data from the <a href="https://archive.org">Internet Archive</a>'s <a href="https://openlibrary.org">OpenLibrary.org</a>
           </p>
         </footer>
 
+        <div class="banner banner-bottom">
+            <div class="banner-content">
+                 <span>This site is a work in progress. To <a href="/volunteer">contribute</a>, please check out our <a href="https://github.com/Open-Book-Genome-Project/TheBestBookOn.com/" target="_blank">Github</a> or  <a href="https://forms.gle/SFmgvBvRF9pAJN2B7" target="_blank">take our survey</a>!</span>
+            </div>
+        </div>
```

Comment:
uBlock Origin is blocking this div because of the `banner-bottom` class.  Probably not a big deal, but worth noting.